plugins/scenario/World.ts

```diff
@@ -22,12 +22,11 @@ export class World {
     // Q: should we really need to fork/snapshot the deployment manager?
     this.hre = hre;
     this.base = base;
-    const networkName = base.name === 'development' ? 'development' : base.network;
+    const networkName = this.isRemoteFork() ? base.network : 'development';
```

Comment:
I thought we agreed in channel not to special case though?

Reply:
deleted